boshrelease/scripts/shipit

```diff
@@ -51,7 +51,7 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 git clone ${REPO_ROOT} ${REPO_OUT}
 
 pushd ${REPO_OUT}
-RELEASE_NAME=$(bosh int config/final.yml --path /final_name)
+RELEASE_NAME=$((set -o pipefail ; bosh int config/final.yml --path /final_name 2> /dev/null | tr -d '\n') || bosh int config/final.yml --path /name)
```

Comment:
What is it you're looking to fix/change here?

Reply:
This tricks enables supporting both `name` as well as `final_name` in `config/final.yml`.

The `| tr -d '\n'` is important so that when we need to pick the `name` property we don't get a spurious newline. But then we need to take into account any `bosh int` failure on the left side of the pipe. Thus we need to adopt the `-o pipefail` Bash option for that.

All-in-all, I didn't find any simpler solution for supporting both `final_name` (the old naming) and `name` (the new one).

`name` is the default when running `bosh init-release`. I've also submitted a PR in BOSH docs recently, in order to advertise about this.

Reply:
Hi Dr Nic,

Coming back to these pipeline templates, I see my pending PRs.

Here, do you want me to suggest something easier to read?

```suggestion
RELEASE_NAME=$(bosh int config/final.yml --path /final_name 2> /dev/null || true)
if [[ -z $RELEASE_NAME ]]; then
    RELEASE_NAME=$(bosh int config/final.yml --path /name)
fi
```

Or this, which is shorter:
```suggestion
RELEASE_NAME=$(bosh int config/final.yml --path /final_name 2> /dev/null) \
    || RELEASE_NAME=$(bosh int config/final.yml --path /name)
```

What do you think about these alternatives?

Benjamin